### ISSUE TITLE
Permanent errorr xul 

### ISSUE THREAD

**Issue by kitaro1** [Feodor2/Mypal#23]
Problems error xul stream tv:
http://wklejto.pl/563034

**Comment by Feodor2** [comment]
You mean online tv?
Why you give me the link of some abracadabra __?xml version="1.0" encoding="UTF-16"?>__ еtс, i not figure how to use it, give me normal link.
So i search any to get tv and did not have the error
http://mywebtv.info/a7-tv

**Comment by kitaro1** [comment]
Yes - it's about tv stream. There is an application error message and the system closes the mypal program.
http://prntscr.com/jurcrg
http://prntscr.com/jurd0u
After closing mypal, the temporary appcompat.txt file disappears.
mywebtv.info works well.
http://prntscr.com/jurfw9 - no polish tv.

**Comment by Feodor2** [comment]
Hey i asked the link, can you give the url where you got the crash, i do not ask something else for now, if i needed that message i probably asked about it.
Of the appcompat.txt i do not recall any time when it was helping so do not bother with it.

**Comment by kitaro1** [comment]
I found the culprit.
In the first screenshot you can see the window message window moved. White background in the message offset - this is a problem with the graphics driver. I downloaded the newest driver from the Intel and after the update the problem ended.
Similarly, I once had a problem in Firefox 33. In this version, support for older graphics drivers has ended.
Similarly - mypal does not support older graphics drivers for several versions. The latest drivers dedicated to the disc were enough. Firefox was working properly. Mypal wanted newer ones ...
